server/hotrod/src/main/java/org/infinispan/server/hotrod/Decoder2x.java

```diff
@@ -411,6 +411,13 @@ public void customReadKey(HotRodHeader header, ByteBuf buffer, CacheDecodeContex
             }).isPresent()) {
                return;
             }
+            if (Constants.isVersionPost25(header.version)) {
+               int listenerInterests = ExtendedByteBufJava.readMaybeVInt(buffer);
+               if (listenerInterests == Integer.MIN_VALUE)
+                  return;
+
+               requestCtx.setListenerInterests(listenerInterests);
+            }
```

Comment:
Is it like [this](https://gist.github.com/galderz/ee907261b68fdebe452b193e075e942f)? 

Also, I didn't see any tests failing.